streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java

```diff
@@ -57,4 +68,147 @@ public void shouldPutAll() {
         assertEquals(store.get(1), "1");
         assertEquals(store.get(2), "2");
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void shouldRemoveLeastUsedEntry() {
```

Comment:
The test seems to be fine, but I am wondering it this really test the `InMemoryLoggedStore` or the wrapped `LruStore` ?

Reply:
It tests the wrapped `LruStore`, but the coverage report counts against `InMemoryKeyValueLoggedStore`, but I'm not married to it so I can remove.